recipe/meta.yaml

```diff
@@ -16,7 +16,7 @@ requirements:
   run:
     - cuda-command-line-tools {{ version }}
     - cuda-visual-tools {{ version }}
-    - gds-tools 1.5.0.59
+    - gds-tools 1.5.0.59                     # [not aarch64]
```

Comment:
Currently there are no builds of `gds-tools` for `linux_aarch64`. So skipping this dependency on that OS/architecture

xref: https://github.com/conda-forge/libcufile-feedstock/issues/6